models/comments-model.js

```diff
@@ -0,0 +1,55 @@
+const format = require("pg-format");
+const db = require("../db/connection");
+exports.selectCommentsByArticleId = (params) => {
+  const id = params.article_id;
+  if (isNaN(id)) {
+    return Promise.reject({ msg: "Bad Request", code: 400, custom: true });
+  }
+
+  let query = `
+  SELECT * FROM comments WHERE article_id = $1
+  ORDER BY created_at DESC;
+  `;
+  const checkQuery = `SELECT * FROM articles WHERE article_id = $1;`;
+
+  return db.query(checkQuery, [id]).then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({ msg: "Not Found", code: 404, custom: true });
+    }
+
+    return db.query(query, [id]).then(({ rows }) => {
+      return rows;
+    });
+  });
+};
+
+exports.insertCommentsByArticleId = (reqBody, params) => {
+  const created_at = new Date(Date.now());
```

Comment:
Don't think this is needed.  Each item when added will auto generate at the moment it is created.  Only need the author, body, and article_id